libdnf/module/module_query.cpp

```diff
@@ -159,4 +160,28 @@ void ModuleQuery::filter_latest(int limit) {
 }
 
 
+void ModuleQuery::filter_nsvca(const Nsvcap & nsvcap, libdnf::sack::QueryCmp cmp) {
+    const std::string & name = nsvcap.get_name();
+    if (name != "") {
```

Comment:
I am sorry, I overlooked it. std::string has method empty that could be used instead. The difference is that the code compare the string form instead of asking whether length inside is 0. If I am correct, std::string knows its length therefore it should be cheaper.

`if (!name.empty()) {`

Reply:
Right, fixed.